cita-executor/core/src/libexecutor/executor.rs

```diff
@@ -388,22 +386,6 @@ impl Executor {
     }
 }
 
-impl<'a> BloomGroupDatabase for Executor {
-    fn blooms_at(&self, position: &GroupPosition) -> Option<BloomGroup> {
-        let position = db_indexes::LogGroupPosition::from(position.clone());
-
-        let log_bloom_group = self
-            .db
-            .get(Some(DataCategory::Extra), &*position.get_index())
-            .map(|bytes| decode::<LogBloomGroup>(bytes.unwrap().as_slice()))
-            .expect("Get bloom group from db error.");
-
-        let bloom_group = log_bloom_group.into();
-
-        Some(bloom_group)
-    }
-}
-
```

Comment:
Another implementation already existed in [cita-chain](https://github.com/cryptape/cita/blob/develop/cita-chain/core/src/libchain/chain.rs#L257). For reviewers.

Reply:
add ours licence ？

Reply:
I think there is no need to add such supplement docs.

In my respective, all the part of license docs in files should be deleted after we finish removing gpl. No matter it is `parity` or `cryptape`